sdk.go

```diff
@@ -3,55 +3,58 @@ package corbado
 import (
 	"net/http"
 
-	"github.com/corbado/corbado-go/pkg/sdk/assert"
-	"github.com/corbado/corbado-go/pkg/sdk/authtoken"
-	"github.com/corbado/corbado-go/pkg/sdk/config"
-	"github.com/corbado/corbado-go/pkg/sdk/emailcode"
-	"github.com/corbado/corbado-go/pkg/sdk/emaillink"
-	"github.com/corbado/corbado-go/pkg/sdk/entity/api"
-	"github.com/corbado/corbado-go/pkg/sdk/passkey"
-	"github.com/corbado/corbado-go/pkg/sdk/project"
-	"github.com/corbado/corbado-go/pkg/sdk/servererror"
-	"github.com/corbado/corbado-go/pkg/sdk/session"
-	"github.com/corbado/corbado-go/pkg/sdk/template"
-	"github.com/corbado/corbado-go/pkg/sdk/user"
-	"github.com/corbado/corbado-go/pkg/sdk/validation"
 	"github.com/pkg/errors"
+
+	"github.com/corbado/corbado-go/pkg/assert"
+	"github.com/corbado/corbado-go/pkg/generated/api"
+	"github.com/corbado/corbado-go/pkg/servererror"
+	"github.com/corbado/corbado-go/pkg/services/authtoken"
+	"github.com/corbado/corbado-go/pkg/services/emailmagiclink"
+	"github.com/corbado/corbado-go/pkg/services/emailotp"
+	"github.com/corbado/corbado-go/pkg/services/passkey"
+	"github.com/corbado/corbado-go/pkg/services/project"
+	"github.com/corbado/corbado-go/pkg/services/session"
+	"github.com/corbado/corbado-go/pkg/services/smsotp"
+	"github.com/corbado/corbado-go/pkg/services/template"
+	"github.com/corbado/corbado-go/pkg/services/user"
+	"github.com/corbado/corbado-go/pkg/services/validation"
 )
 
-const Version = "v0.6.0"
+const Version = "1.0.0"
 
 type SDK interface {
 	AuthTokens() authtoken.AuthToken
-	EmailCodes() emailcode.EmailCode
-	EmailLinks() emaillink.EmailLink
+	EmailOTPs() emailotp.EmailOTP
+	EmailMagicLinks() emailmagiclink.EmailMagicLink
 	Passkeys() passkey.Passkey
 	Projects() project.Project
 	Sessions() session.Session
 	Templates() template.Template
 	Users() user.User
 	Validations() validation.Validation
+	SmsOTPs() smsotp.SmsOTP
 }
 
 type Impl struct {
 	client     *api.ClientWithResponses
 	HTTPClient *http.Client
 
-	authTokens authtoken.AuthToken
-	emailCodes emailcode.EmailCode
-	emailLinks emaillink.EmailLink
-	passkeys   passkey.Passkey
-	projects   project.Project
-	sessions   session.Session
-	templates  template.Template
-	validation validation.Validation
-	users      user.User
+	authTokens      authtoken.AuthToken
+	emailOTPs       emailotp.EmailOTP
+	emailMagicLinks emailmagiclink.EmailMagicLink
+	passkeys        passkey.Passkey
+	projects        project.Project
+	sessions        session.Session
+	templates       template.Template
+	validations     validation.Validation
+	users           user.User
+	smsOTPs         smsotp.SmsOTP
```

Comment:
put `smsOTPs` between `sessions` and `templates` to keep them alphabetically sorted.